tests/secrets.sh

```diff
@@ -52,7 +52,7 @@ SECRETKEY=$(docker secret create test-secrets-key  /tmp/test-secrets-key.pem)
 DOCKERSERVICE=$(docker service create -q -d -p "${CLIENT_PORT}:6667" -p "${TLS_CLIENT_PORT}:6697" --secret source=test-secrets-key,target=inspircd.key --secret source=test-secrets-cert,target=inspircd.crt inspircd:testing)
 sleep 35
 # Make sure TLS is working
-TLSCHECK=$(echo quit | timeout 10 openssl s_client -ign_eof -connect "localhost:${TLS_CLIENT_PORT}" 2>/dev/null | grep -c "OU=Secret Server Admins")
+TLSCHECK=$(echo quit | timeout 10 openssl s_client -ign_eof -connect "$(cat /etc/hostname):${TLS_CLIENT_PORT}" 2>/dev/null | grep -c "OU\s*=\s*Secret Server Admins")
```

Comment:
Careful, a hostname has to be configured **correctly** in order to resolve to an IP address. That's not always the case. I would keep that on localhost instead.

Reply:
You are certainly correct. I am just having issues with this test on my local machine where connecting openssl through localhost hangs after connect, but works fine if the hostname or non-loopback address is used:
```console
$ openssl s_client -ign_eof -connect localhost:31142 -security_debug
CONNECTED(00000003)
# hangs here
```
I figured it was normal docker swarm behavior, but I guess not, since the test works fine for travis. Anyway, I made the fixed up the requested change. The build will still fail though, because of the release of v3.2.0, but https://travis-ci.org/inspircd/inspircd-docker/builds/568864531?utm_source=github_status&utm_medium=notification showed that the build should pass otherwise (when I disable 'fail on newer release available').